webcompat/static/js/lib/issues.js

```diff
@@ -61,7 +77,17 @@ var defaultLinkOpenRender =
   };
 
 md.renderer.rules.link_open = function(tokens, idx, options, env, self) {
-  tokens[idx].attrPush(["rel", "nofollow"]);
+  var foundReportedURL = checkIfReportedURLOpeningA(tokens);
+  // only add target=_blank to a URL that was parsed from the following:
+  // **URL**: <url>
+  if (foundReportedURL) {
+    tokens[idx].attrPush(["rel", "nofollow"]);
+    tokens[idx].attrPush(["target", "_blank"]);
+  } else {
+    tokens[idx].attrPush(["rel", "nofollow"]);
```

Comment:
Adding this line once before or after the if?

Reply:
+1